src/lsptoolshost/fixAllCodeAction.ts

```diff
@@ -24,10 +24,14 @@ export function registerCodeActionFixAllCommands(
 }
 
 export async function getFixAllResponse(
-    data: LSPAny,
+    data: RoslynProtocol.CodeActionResolveData,
     languageServer: RoslynLanguageServer,
     outputChannel: vscode.OutputChannel
 ) {
+    if (!data.FixAllFlavors) {
```

Comment:
nit: check the uniqueID resolves too?

Reply:
fixallflavors is a nullable property on the data field, but uniqueId is not, we shouldn't need to verify it again